### DISCUSSION TITLE
求移植,不要Wine 植物大战僵尸

### DISCUSSION THREAD

**Discussion by erroreutopia** [Gamuxorg/bbs#422] [投递和请求游戏]
有大佬移植一下吗？,不要wine的

**Comment by shouhuanxiaoji** [comment]
该产品源码开放了嘛？

**Comment by nukemiko** [comment]
植物大战僵尸不开放源码的吧，那就没戏，继续用wine将就吧

**Comment by dfh1949** [comment]
https://github.com/ErLinErYi/PlantsVsZombies
开源项目

**Reply by erroreutopia** [reply]
但是Linux上似乎难以编译啊--至少我不会😢

**Comment by kero990** [comment]
> 有大佬移植一下吗？,不要wine的

现在倒是有个曲线救国的方案
植物大战僵尸在国内由拓维代理，现在他们出了一个微信小程序版的pvz1，还原度很高
而且现在的linux版微信可以非常丝滑的运行小程序，所以你只要在微信里玩就可以了，完全不输原生的性能。
可以参见我在deepin论坛的帖子
https://bbs.deepin.org/zh/post/272294
